kvm/kvm.go

```diff
@@ -137,3 +140,62 @@ func Run(vcpuFd uintptr) error {
 func GetVCPUMMmapSize(kvmFd uintptr) (uintptr, error) {
 	return Ioctl(kvmFd, IIO(kvmGetVCPUMMapSize), uintptr(0))
 }
+
+// InitKVM takes care of the general kvm setup without dependencies to runtime target.
+func InitKVM(kvmPath string, tssaddr, identmapaddr uint32, nCpus int) (uintptr, uintptr, []uintptr, []*RunData, error) {
```

Comment:
I want the kvm package to simply provide the KVM API as a wrapper for ioctl. If `machine.New()` seems complicated, how about implementing a private function (e.g. `machine.initVMandVCPUs()`) in the machine package?